airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml

```diff
@@ -0,0 +1,378 @@
+version: 0.65.0
+type: DeclarativeSource
+
+definitions:
+  schema_loader:
+    type: JsonFileSchemaLoader
```

Comment:
can we move the schema files to the manifest and replace the schema loader with a `InlineSchemaLoader`?

Reply:
Sure, should the schema files be deleted after I replace the schema loader with InlineSchemaLoader or should I keep the schema files that are there in the schemas folder?

Reply:
yes let's delete them

Reply:
the context is that defining the schemas in-line allows the connector builder to load the full connector definition

Reply:
Ah, that makes sense, I'll use an InlineSchemaLoader now onwards for migrations.